### ISSUE TITLE
TreeView SelectedItem not working

### ISSUE THREAD

**Issue by KeKl** [AvaloniaUI/Avalonia#814]
Hi,

the code looks like:

```
<TreeView Items="{Binding Topics}" SelectedItem="{Binding SelectedTopic}" />
```
The viewmodel is derived from ReactiveObject:

```
public ObservableCollection<string> Topics { get; private set; }
string _selectedTopic;
public string SelectedTopic
{
    get { return _selectedTopic; }
    set { this.RaiseAndSetIfChanged(ref _selectedTopic, value); }
}
```

The _SelectedItem_ will never be set. This works for ListBox. I use v0.4.0.

**Comment by danwalmsley** [comment]
@KeKl as a workaround until this gets fixed please try:

SelectedItem="{Binding SelectedTopic, Mode=TwoWay}">

**Comment by danwalmsley** [comment]
I have submitted a PR to resolve this. #815

**Comment by KeKl** [comment]
Thanks thats working

**Comment by grokys** [comment]
Closing this because I believe the correct fix is #830
